.github/workflows/tests_sc_p4est.yml

```
@@ -103,7 +103,7 @@ jobs:
       run: echo No cache found or cache will be ignored. SC_IGNORE_CACHE=$SC_IGNORE_CACHE
     - name: if ignore cache, delete folders
       if: ${{ env.SC_IGNORE_CACHE == 1 }}
-      # The true at the end is to ignore errors that i.e. occur when the folders do not exist
+      # The true at the end is to ignore errors that i.e. cadur when the folders do not exist
```

@jfussbro This is not good ... please check for this before making a PR. Most text substitution tools have a parameter/switch where you tell it to only replace stand-alone words. What tool are you using?

```suggestion
      # The true at the end is to ignore errors that i.e. occur when the folders do not exist
```

---

I'm sorry. I didn't think about that. I used find and replace in VS code.